src/Raven.Studio/wwwroot/App/views/resources/databases.html

```diff
@@ -247,7 +247,15 @@ <h2 class="text-center text-muted">No databases have been created</h2>
                                                 <i class="icon-play"></i> Enable indexing
                                             </a>
                                         </li>
-                                        <li class="divider" data-bind="visible: $root.createIsLocalDatabaseObservable(name) &&  $root.accessManager.canCompactDatabase()"></li>
+                                        <li class="divider" data-bind="visible: $root.createIsLocalDatabaseObservable(name) && ($root.accessManager.canCompactDatabase() || $root.accessManager.canRestartDatabase())"></li>
+                                        <li data-bind="visible: $root.createIsLocalDatabaseObservable(name)() && $root.accessManager.canRestartDatabase()">
+                                            <a data-bind="visible: disabled" title="The database is disabled" class="has-disable-reason disabled" data-placement="top">
+                                                <i class="icon-reset"></i> Restart database
+                                            </a>
+                                            <a href="#" data-bind="click: $root.restartDatabase, visible: !disabled()">
+                                                <i class="icon-reset"></i> Restart database
```

Comment:
@grisha-kotler 
IMHO (and if I understand correctly)
Must add a tooltip/explain that this restart is only for the current/local node